src/services/WolfDen.Domain/Entity/LeaveRequest.cs

```diff
@@ -0,0 +1,40 @@
+﻿using WolfDen.Domain.Enum;
+
+namespace WolfDen.Domain.Entity
+{
+    public class LeaveRequest
+    {
+        public int Id { get; private set; }
+        public int EmployeeId { get; private set; }
+        public virtual Employee Employee { get; private set; }
+
+        public int TypeId { get; private set; }
+        public virtual LeaveType LeaveType { get; private set; }
+
+        public bool HalfDay { get; private set; }
+        public DateOnly FromDate { get; private set; }
+        public DateOnly ToDate { get; private set; }
+        public DateOnly ApplyDate { get; private set; }
+        public LeaveRequestStatus LeaveRequestStatus { get; private set; }
+        public string Description { get; private set; }
```

Comment:
is this mandatory?

Reply:
To make Post operation depend on Model Class and to restrict it from setting values from outside.